indexer/src/types/transaction.ts

```diff
@@ -28,35 +26,69 @@ import {
   TxValuesArray,
   TypedTransaction,
   TypedTxData,
+  hexToBytes,
+  bytesToBigInt,
+  JsonTx,
 } from "../deps.ts";
 
+type HexString = `0x${string}`
+
+interface ToEthTxRequest {
+  transaction: Transaction;
+  receipt: TransactionReceipt;
+  blockNumber: PrefixedHexString;
+  blockHash: PrefixedHexString;
+  isPendingBlock: boolean;
+}
```

Comment:
In all the interfaces like this one, can we have a header documentation for the interface as well as for the fields inside in order to document everything properly and for a new user to understand each line of the code

Reply:
added headers documentation and fields description